src/app/api/rbi/route.ts

```diff
@@ -0,0 +1,39 @@
+import { NextResponse } from 'next/server'
+import { pool } from '@/lib/db'
+
+const query = `
```

Comment:
I wondering if we should use a query builder to simplify the sql code maintainability. WDYT?

Reply:
I know we may lose in performance, but probably it's going to be better in the long run